### PR TITLE
CDSK-812 - add revision column to MyBuilds

### PR DESCRIPTION
**Pull request by warcholprzemo**
Add revision column  every (codebase / branch / short_revision) into MyBuilds page.

### CODE REVIEW

**Comment by warcholprzemo** [comment]
revision column on MyBuilds with linked repo and commit

revision_column

**Comment by campos-ddc** [comment]
Shouldn't your screenshot say the branch name instead of `HEAD`?

**Comment by warcholprzemo** [comment]
@campos-ddc This is just my local configuration in master.cfg.
If I choose for example `master` it will be shown.

master_branch

**Review by campos-ddc — APPROVED** [review]

File: master/buildbot/status/web/mybuilds.py [hunk 1]
```diff
@@ -10,29 +11,51 @@ class MybuildsResource(HtmlResource):
...
+        return display_repositories
+
+    @staticmethod
+    def prepare_builds_by_ssid(builds):
+        builds_by_ssid = {}
+        for row in builds:
+            row['sourcestamps'] = []
+            row['query_params'] = []
+            builds_by_ssid[row['sourcestampsetid']] = row
+        return builds_by_ssid
```
Comment (Mowinski) [inline master/buildbot/status/web/mybuilds.py]: Please add the new line.

**Review by Mowinski — APPROVED** [review]
Please resolve conflict

File: master/buildbot/status/web/mybuilds.py (hunk 1 above)
Comment (warcholprzemo) [reply master/buildbot/status/web/mybuilds.py]: Done

**Pull request merged**